src/components/CameraControls.tsx

```diff
@@ -32,9 +32,9 @@ export default function CameraControls(props: any) {
       dampingFactor={0.08}
       screenSpacePanning
       mouseButtons={{
-        LEFT: THREE.MOUSE.RIGHT,
-        MIDDLE: THREE.MOUSE.MIDDLE,
-        RIGHT: THREE.MOUSE.LEFT,
+        LEFT: THREE.MOUSE.ROTATE,
+        MIDDLE: THREE.MOUSE.DOLLY,
+        RIGHT: THREE.MOUSE.PAN,
```

Comment:
nice, are these just aliases?

Reply:
yeah, basically. I think sometimes left/right are flipped in terms of which is for rotation vs panning so I just wanted to be explicit